### PR TITLE
Jira 347 csp new drug submission information

### PR DESCRIPTION
**Pull request by tfang54321**

### CODE REVIEW

**Review by AMckayHC — CHANGES_REQUESTED** [review]
Can you update the French JSON file? You can find the french text here: https://health-products.canada.ca/forms/certificate2023/formulaire-certificat-protection-supplementaire-2023.html

**Comment by tfang54321** [comment]
@AMckayHC  I have translate some csp label to french. and create new build script for csp. please review, approve and merge if everything looks good.  thanks

File: apps/csp/build-content-configs/metaData-en.html
Comment (AMckayHC) [inline apps/csp/build-content-configs/metaData-en.html:1]: Need to change metadata to be what is in CSP Production.

File: apps/csp/build-content-configs/metaData-fr.html
Comment (AMckayHC) [inline apps/csp/build-content-configs/metaData-fr.html:1]: Need to change metadata to be what is in CSP Production.

**Review by AMckayHC — COMMENTED** [review]
Can you change breadcrumbs to be copied over from CSP, thanks!

File: apps/csp/build-content-configs/metaData-fr.html
Comment (tfang54321) [reply apps/csp/build-content-configs/metaData-fr.html:1]: changes made. please review.

**Review by AMckayHC — APPROVED** [review]

**Pull request merged**